csv.cpp

```diff
@@ -258,3 +258,17 @@ void remove_bom(std::fstream& stream){
         stream.unget();
     }
 }
+
+/**
+ * @brief check if row is empty
+ * Avoid to process empty row and row with only carriage return characters.
+ *
+ * @return true if row is empty
+ */
+bool CsvReader::row_is_empty(const std::vector<std::string>& row) {
+    if(row.empty() || (row.size() == 1 && row[0].empty()))
+        return true;
+    else
```

Comment:
minor: most of the time we use brackets `} else {`

Reply:
:white_check_mark: